plugin.video.iplayerwww/default.py

```diff
@@ -0,0 +1,243 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import division
+
+import os
+import sys
+import urllib
+
+import xbmc
+import xbmcaddon
+import xbmcgui
+import xbmcplugin
+
+
+plugin_handle = int(sys.argv[1])
+ADDON = xbmcaddon.Addon(id='plugin.video.iplayerwww')
+sys.path.insert(0, os.path.join(ADDON.getAddonInfo("path"), 'resources', 'lib'))
```

Comment:
please use relative module imports instead of sys.path.append:

```
from resources.lib import whateveryouneed
```

Reply:
Should be fixed.